internal/devicescale/devicescale.go

```diff
@@ -46,3 +46,16 @@ func GetAt(x, y int) float64 {
 
 	return s
 }
+
+// ScreenScaleAt returns the screen scale at (x, y), i.e. the number of video mode pixels per device-dependent pixel.
```

Comment:
As this is used only for video mode, what about renaming this at `VideoModeScaleAt`? The term 'screen' is not clear here.

Reply:
Good idea, done.